flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2.java

```diff
@@ -337,4 +337,188 @@ private StructLikeSet actualRowSet(long snapshotId, String... columns) throws IO
     }
     return set;
   }
+
+  @Test
+  public void testChangeLogOnIdKeyWithSnapshotExpire() throws Exception {
```

Comment:
Since this is updating a file in core, I would expect the tests to be more specific and located in core, not in the Flink integration.